src/components/tab/tab.component.ts

```diff
@@ -50,7 +50,11 @@ export default class SlTab extends ShoelaceElement {
   /** Disables the tab and prevents selection. */
   @property({ type: Boolean, reflect: true }) disabled = false;
 
-  tabIndex = -1;
+  /**
+   * @internal
+   * Need to wrap in a `@property()` otherwise NextJS blows up.
```

Comment:
```suggestion
   * Need to wrap in a `@property()` otherwise CustomElement throws a "The result must not have attributes" runtime error.
```

Not specific to Nextjs, replicated this behavior in React SPA & Vue too!

Reply:
😩 Seems we should probably add a lint rule for this 🤔